app/resources/file_sets/file_sets_controller.rb

```diff
@@ -53,6 +54,14 @@ def update
     after_update_error error
   end
 
+  def after_delete_success
+    redirect_to solr_document_path(@parent_resource)
+  end
+
+  def parent_resource
+    @parent_resource ||= find_resource(params[:id]).decorate.parent
```

Comment:
I think we should prefer `Wayfinder.for(file_set).parents` over using the decorator.

It might be nice to extract a method while we're at it like
```
def file_set
  @file_set ||= find_resource(params[:id])
end
```
Since that `find_resource` is called so much.

Reply:
ok I'll use the Wayfinder!